gdraw/ggdkdraw.c

```diff
@@ -1221,6 +1247,10 @@ static void GGDKDrawSetDefaultIcon(GWindow icon) {
     GGDKWindow gicon = (GGDKWindow)icon;
     if (gicon->is_pixmap) {
         gicon->display->default_icon = gicon;
+#ifdef FONTFORGE_CAN_USE_GTK_BRIDGE
```

Comment:
Grab an icon to use for window decoration. Sort of works now but the image comes out funny on my (KDE Plasma) system.